FHIR-41543 and FHIR-41534

yanheras opened the pull request:
Applied all changes except updating the IG dependency from USCore 5.0.0 to USCore 6.1.0. (Not able to update because it breaks the build due to changes required for other profiles)
Once the dependency is updated to USCore 6.1.0, the following changes should be reflected in addition to what are changed in this pull request, which should complete all changes stated in this tracker:
Derive from US Core 6.1.0 QuestionnaireResponse profile
meta section – remove as this was part of US Core 5.0 and is not present in US Core 6.1.0
The snapshot view will show value[x] valueDecimal, valueString, valueCoding as MS from USCore

abdullah-git1 reviewed (changes requested):
For item 6 in the [Jira ticket](https://jira.hl7.org/browse/FHIR-41543)

The following seemed to be missing/incomplete:

QI-Core QuestionnaireResponse.value[x] – US Core includes options valueDecimal, valueString, valueCoding. **RETAIN QI-Core options valueDate, valueDateTime, valueDecimal, valueInteger, valueString, valueCoding, valueQuantity**

yanheras commented:
Updated and added the missing valueDecimal, valueString, and valueCoding that are from the USCore profile. The value[x] data types should include all now.

abdullah-git1 reviewed (approved):
looks good
